public/heatmap.html

```diff
@@ -13,6 +14,7 @@
 		<md-toolbar>
 			<div class="md-toolbar-tools">
 				<h2 flex md-truncate>Ocelot Analytics</h2>
+				<a href="Travel.html">View Pathing Analytics</a>
```

Comment:
Instead of this button, we should just make different paths go to different pages (rtgames.rtaustin.local:8080/heatmap and rtgames.rtaustin.local:8080/travel). I can make that change if you remove the edits to this file